config/prow/cluster/cherrypick_deployment.yaml

```diff
@@ -0,0 +1,80 @@
+kind: List
```

Comment:
List may not work in the k8s, it is actually a openshift constraint,.. lets devide these resources with seperator `---` like we did in the other files.